src/cluster_manager.sh

```diff
@@ -0,0 +1,405 @@
+#!/bin/bash
+# MicroShift Cluster Manager
+# Primitive functions for managing MicroShift clusters
+
+set -euo pipefail
+
+# Configuration - These variables can be overridden by the environment
+# They are used throughout the script for single-node and multi-node cluster management
+USHIFT_MULTINODE_CLUSTER="${USHIFT_MULTINODE_CLUSTER:-microshift-okd-multinode}"
+NODE_BASE_NAME="${NODE_BASE_NAME:-microshift-okd-}"
+USHIFT_IMAGE="${USHIFT_IMAGE:-microshift-okd}"
+LVM_DISK="${LVM_DISK:-/var/lib/microshift-okd/lvmdisk.image}"
+LVM_VOLSIZE="${LVM_VOLSIZE:-1G}"
+VG_NAME="${VG_NAME:-vg-${USHIFT_MULTINODE_CLUSTER}}"
```

Comment:
TopoLVM currently expects a hard-coded `myvg1` volume name - how does this work?